app/models/pledge_config.rb

```diff
@@ -1,11 +1,6 @@
 class PledgeConfig < ApplicationRecord
   acts_as_tenant :fund
 
-  validates_presence_of :contact_email,
```

Comment:
different fields now do this so there's a legit reason for these not to be filled out